### PR TITLE
Popova test consumer cart

### PR DESCRIPTION
**Pull request by AlexandraPopovaz**
New PR

### CODE REVIEW

**Review by WadeAnisimov — APPROVED** [review]

File: cypress/integration/consumersCart.spec.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ it('Add one item to the cart', () => {
   const loginPage = new LoginPage();
   
   loginPage.openPage()
-  loginPage.userName().type(username);
+  loginPage.username().type(username);
```
Comment (ghost) [inline cypress/integration/consumersCart.spec.js]: please follow` camelCase` path `userName`

Comment (MaximChetverikov) [inline cypress/integration/consumersCart.spec.js]: please follow `camelCase` path `userName`

Comment (AlexandraPopovaz) [reply cypress/integration/consumersCart.spec.js]: Thanks! Fixed

Comment (AlexandraPopovaz) [reply cypress/integration/consumersCart.spec.js]: Thanks! Fixed

**Review by MaximChetverikov — APPROVED** [review]

**Pull request merged**